worlds/witness/WitnessItems.txt

```diff
@@ -18,16 +18,22 @@ Progression:
 200 - Progressive Dots - Dots,Full Dots
 260 - Progressive Stars - Stars,Stars + Same Colored Symbol
 
-Usefuls:
-101 - Functioning Brain - False
-510 - Puzzle Skip - True
+Useful:
+510 - Puzzle Skip
+#511 - Energy Capacity
```

Comment:
If your IDE supports it, probably install a PEP8 plugin :) That's the code style for the Archipelago repo and I'm pretty sure this line violates PEP8